libs/utils/include/utils/Panic.h

```diff
@@ -428,6 +428,10 @@ class ArithmeticPanic : public TPanic<ArithmeticPanic> {
 #   define PANIC_FILE(F) ""
 #endif
 
+#if _MSC_VER
```

Comment:
put this in `utils/compiler.h` also maybe assert that __PRETTY_FUNCTION__ isn't defined

Reply:
done, thanks. Instead of asserting I define the macro only if it is not already defined.